### PR TITLE
Add github issue template

### PR DESCRIPTION
**Pull request by StevenLangbroek**
I took the issue template from the `styled-components` core repo, and made some minor tweaks (instructing users to add their stylelint configs to the `envinfo` command. 

Is there anything else that'd be useful info to have? `.stylelintrc`?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+What will it allow you to do that you can't do today?
+How will it make current work-arounds straightforward?
+What potential bugs and edge cases does it help to avoid?
+Please keep it product-centric.
+-->
+
+<!-- BUG TEMPLATE -->
+## Environment
+<!-- Please add your stylelint configurations to the `npmPackages` below, run this command inside your project and paste its contents here (it automatically copies to your clipboard) -->
+`npx envinfo --system --binaries --npmPackages styled-components,babel-plugin-styled-components,stylelint-processor-styled-components, --markdown --clipboard`
```
Comment (chinesedfan) [inline .github/ISSUE_TEMPLATE.md]: Let's add packages mentioned in README.md and how about removing `babel-plugin-styled-components`? Then it is, `--npmPackages stylelint, styled-components, stylelint-processor-styled-components, stylelint-config-styled-components, stylelint-config-recommended`.

And of course, `.stylelintrc` is welcome to be provided.

Comment (StevenLangbroek) [reply .github/ISSUE_TEMPLATE.md]: Good point!

**Comment by StevenLangbroek** [comment]
There you go :v: hope it helps!

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ Please keep it product-centric.
 <!-- BUG TEMPLATE -->
 ## Environment
 <!-- Please add your stylelint configurations to the `npmPackages` below, run this command inside your project and paste its contents here (it automatically copies to your clipboard) -->
-`npx envinfo --system --binaries --npmPackages styled-components,babel-plugin-styled-components,stylelint-processor-styled-components, --markdown --clipboard`
+`npx envinfo --system --binaries --npmPackages --npmPackages stylelint,styled-components,stylelint-processor-styled-components,stylelint-config-styled-components,stylelint-config-recommended --markdown --clipboard`
```
Comment (chinesedfan) [inline .github/ISSUE_TEMPLATE.md]: Oops, you pasted 2 `--npmPackages`. 😆

Comment (StevenLangbroek) [reply .github/ISSUE_TEMPLATE.md]: 🤦‍♂️

Comment (StevenLangbroek) [reply .github/ISSUE_TEMPLATE.md]: Shit sorry totally spaced on this. Fixing now!

**Comment by mxstbr** [comment]
Thank you so much for helping us improve styled-components! Based on our [Community Guidelines](https://github.com/styled-components/styled-components/blob/master/CONTRIBUTING.md) every person that has a PR of any kind merged is offered **an invitation to the styled-components organization**—that is you right now!

Should you accept, you'll get write access to the main repository. (and a fancy `styled-components` logo on your GitHub profile!) You'll be able to label and close issues, merge pull requests, create new branches, etc. We want you to help us out with those things, so don't be scared to do them! Make sure to read our [contribution and community guidelines](https://github.com/styled-components/styled-components/blob/master/CONTRIBUTING.md), which explains all of this in more detail. Of course, if you have any questions just let me know!

**Pull request merged**